### ISSUE TITLE
payday 2 "error:beardlib_version"

### ISSUE THREAD

**Issue by K-estzoo77** [simon-wh/PAYDAY-2-BeardLib#745]
I don't know what happen but when i go to see all my mod in my game it's write on every mod and beardlib itself "error:beardlib_version" i try to reinstall beardlib with different version of it but nothing change.
Someone know what happen and what to do for resolve it ?

![218620_20240930175809_1](https://github.com/user-attachments/assets/efefdd34-8cd5-4a14-9376-a692aa00c842)

**Comment by sexoman1** [comment]
happening the same here

**Comment by Bobthebuilder414** [comment]
Anyone found the problem fix?

**Comment by ORLOX93** [comment]
> I know this is old, but what version of beardlib are you using? I’ve not had this happen

I have the same issue, but i play the game on spanish.
Could it be that this problem doesnt happen in english?

**Comment by K-estzoo77** [comment]
personally i always have the ERROR beardlib_version but i don't know why all my mods works good.
so i think you just need to ignore the bug and try mod normally. 
since september i play with many mod and everything work despite the ERROR beardlib_version

**Comment by ORLOX93** [comment]
> personally i always have the ERROR beardlib_version but i don't know why all my mods works good. so i think you just need to ignore the bug and try mod normally. since september i play with many mod and everything work despite the ERROR beardlib_version

Yeah, I have had the error since I started using mods in december of last year, it doesn't seem to affect anything, but also I have no way of verifying if it does or not.
